framework/encode/vulkan_capture_manager.cpp

```diff
@@ -1889,6 +1889,59 @@ void VulkanCaptureManager::PreProcess_vkCreateSwapchain(VkDevice
     }
 }
 
+void VulkanCaptureManager::PostProcess_vkCreateSwapchainKHR(VkResult                        result,
+                                                            VkDevice                        device,
+                                                            const VkSwapchainCreateInfoKHR* pCreateInfo,
+                                                            const VkAllocationCallbacks*    pAllocator,
+                                                            VkSwapchainKHR*                 pSwapchain)
+{
+    GFXRECON_UNREFERENCED_PARAMETER(device);
+    GFXRECON_UNREFERENCED_PARAMETER(pAllocator);
+    GFXRECON_UNREFERENCED_PARAMETER(pSwapchain);
+
+    GFXRECON_ASSERT(pCreateInfo != nullptr);
+
+    if (pCreateInfo)
```

Comment:
I think this if is unnecessary

Reply:
Fair, we have the assert. Even if we stop from crashing here, it will surely crash in the driver!

Reply:
Fixed